crashsimilarity/crash_similarity.py

```diff
@@ -243,8 +245,15 @@ def signature_similarity(model, paths, signature1, signature2):
 if __name__ == '__main__':
     # download_data.download_crashes(days=7, product='Firefox')
     # paths = download_data.get_paths(days=7, product='Firefox')
-    paths = ['../crashsimilarity_data/firefox-crashes-2016-11-09.json.gz', '../crashsimilarity_data/firefox-crashes-2016-11-08.json.gz', '../crashsimilarity_data/firefox-crashes-2016-11-07.json.gz', '../crashsimilarity_data/firefox-crashes-2016-11-06.json.gz', '../crashsimilarity_data/firefox-crashes-2016-11-05.json.gz', '../crashsimilarity_data/firefox-crashes-2016-11-04.json.gz', '../crashsimilarity_data/firefox-crashes-2016-11-03.json.gz']
+    paths = ['../crashsimilarity_data/firefox-crashes-2016-11-09.json.gz',
```

Comment:
> But again the same question here: why should we care about code that will be completely removed in the very resent(!) future?

It's better to keep a clean history for the future (both for the people who are going to look at the code in the future and for finding regressions). When you look at the history you don't want to see unrelated changes, but want every commit to be clean.